packages/site/src/data/solidjs/blogs.ts

```diff
@@ -4,11 +4,30 @@ export const blogTags = [] as const
 
 export const blogs: Blog<typeof blogTags[number]>[] = [
 	{
-	  title: "test",
-	  author: "test",
-	  description: "test.",
-	  image: "https://github.com/uidotdev.png",
-	  href: "https://ui.dev/blog/",
-	  tags: []
+		title: "Ryan Carniatos' blog",
```

Comment:
Should this entry have a different title? Maybe something like "Ryan Carniato on dev.to"